packages/common/src/core/schemas/hubproject.ts

```diff
@@ -36,6 +36,12 @@ export const HubProjectSchema: IConfigurationSchema = {
         featuredImage: {
           type: "object",
         },
+        relatedItems: {
```

Comment:
Sorry I think I might have been incorrect earlier when I said you could call this `relatedItems`. We'll actually need the property defined on the schema to match the property we're storing on the item, so we'll want to make this `relatedItemIds`